test/integration_test/snapshot_test.go

```diff
@@ -134,13 +138,18 @@ func groupSnapshotTest(t *testing.T) {
 }
 
 func groupSnapshotScaleTest(t *testing.T) {
-	ctxsToDestroy := make([]*scheduler.Context, 0)
+	alllContexts := make([]*scheduler.Context, 0)
 	// Triggers 2 snaps, so use half the count in the loop
 	for i := 0; i < snapshotScaleCount/2; i++ {
 		ctxs := createGroupsnaps(t)
-		ctxsToDestroy = append(ctxsToDestroy, ctxs...)
+		alllContexts = append(alllContexts, ctxs...)
 	}
-	destroyAndWait(t, ctxsToDestroy)
+
+	for _, ctx := range alllContexts {
+		verifyGroupSnapshot(t, ctx, defaultWaitTimeout)
```

Comment:
The timeout needs to be increased if there are a large number of snapshots, similar to `snapshotScaleTest`

Reply:
done I'm also doing that when it's equal to 10 since I just had a run where the default timeout wasn't sufficient for scale factor = 10